admin_guide/manage_nodes.adoc

```diff
@@ -146,76 +156,118 @@ Pods are not scheduled for the node until the node passes the health checks that
+For example:
 
-****
-`$ oadm manage-node _<node1>_ _<node2>_ --schedulabe` or
+====
+[options="nowrap"]
+----
+$ oadm manage-node node1.example.com --schedulable=false
+NAME                 LABELS                                        STATUS
+node1.example.com    kubernetes.io/hostname=node1.example.com      Ready,SchedulingDisabled
+----
+====
+
+To mark a currently unschedulable node or nodes as schedulable:
 
-`$ oadm manage-node _<node1>_ _<node2>_ --schedulabe=true`
-****
+----
+$ oadm manage-node <node1> <node2> --schedulabe
+----
 
-Marking node as schedulable will allow any new pods to be scheduled on the
-node.
+Or:
+
+----
+$ oadm manage-node <node1> <node2> --schedulabe=true
+----
 
-Instead of specifying the nodes _<node1>_ _<node2>_, you can also use
---selector=_<node_selector>_ to mark selected nodes as schedulabe or
-unschedulable.
+Alternatively, instead of specifying specific node names (e.g., `_<node1>_
+_<node2>_`), you can use the `--selector=_<node_selector>_` option to mark
+selected nodes as schedulabe or unschedulable.
 
 == Evacuating Pods on Nodes
-Use the following command to evacuate all or selected pods on one or more nodes:
+Nodes must first be link:#marking-nodes-as-unschedulable-or-schedulable[marked
+unschedulable] to perform pod evacuation. Only pods backed by a
+link:../architecture/core_objects/kubernetes_model.html#replication-controller[replication
+controller] are evacuated. Bare pods are unaffected by default.
 
-****
-`$ oadm manage-node _<node1>_ _<node2>_ --evacuate [--pod-selector=_<pod_selector>_]`
-****
+To list pods that will be migrated without actually performing the evacuation,
+use the `--dry-run` option:
 
-Nodes must be marked unschedulable to perform pod evacuation. Only pods backed
-by replication controller will be evacuated. Bare pods will not be touched.
-You can force deletion of bare pods by using --force option:
+----
+$ oadm manage-node <node1> <node2> \
+--evacuate --dry-run [--pod-selector=<pod_selector>]
+----
 
-****
-`$ oadm manage-node _<node1>_ _<node2>_ --evacuate --force [--pod-selector=_<pod_selector>_]`
-****
+To actually evacuate all or selected pods on one or more nodes:
 
-You can list pods that will be migrated by using --dry-run option:
+----
+$ oadm manage-node <node1> <node2> \
+--evacuate [--pod-selector=<pod_selector>]
+----
 
-****
-`$ oadm manage-node _<node1>_ _<node2>_ --evacuate --dry-run [--pod-selector=_<pod_selector>_]`
-****
+You can force deletion of bare pods by using the `--force` option:
+
+----
+$ oadm manage-node <node1> <node2> \
+--evacuate --force [--pod-selector=<pod_selector>]
+----
 
-Instead of specifying the nodes _<node1>_ _<node2>_, you can also use
---selector=_<node_selector>_ to evacuate pods on selected nodes.
+Alternatively, instead of specifying specific node names (e.g., `_<node1>_
+_<node2>_`), you can use the `--selector=_<node_selector>_` option to evacuate
+pods on selected nodes.
```

Comment:
Perhaps you can refer to the previously expressed "--selector" idea.  Actually, if --selector is common to many commands (or invocations), you might consider factoring it an intro section and then defining an abstract NODE-REF that expands to either specific node names or to --selector=SELECTOR.  Then all the examples can say NODE-REF (or something like that).